website/content/en/status/report-2023-07-2023-09/dpaa2.adoc

```diff
@@ -0,0 +1,36 @@
+=== NXP DPAA2 support
+
+Links +
+link:https://cgit.freebsd.org/src/tree/sys/dev/dpaa2[DPAA2 in the FreeBSD source tree] +
+link:https://github.com/mcusim/freebsd-src[DPAA2 on Github]
```

Comment:
```suggestion
Links: +
link:https://cgit.freebsd.org/src/tree/sys/dev/dpaa2[DPAA2 in the FreeBSD source tree] URL: link:https://cgit.freebsd.org/src/tree/sys/dev/dpaa2[] +
link:https://github.com/mcusim/freebsd-src[DPAA2 on Github] URL: link:https://cgit.freebsd.org/src/tree/sys/dev/dpaa2[]
```

Whilst the repetitions, prefixes etc. do appear peculiar, some peculiarities are currently required for the eventual plain text edition of the report to include visible or usable links. See, for example, last quarter's:

* https://lists.freebsd.org/archives/freebsd-announce/2023-July/000077.html